src/main/java/edu/harvard/iq/dataverse/util/json/JsonPrinter.java

```diff
@@ -331,6 +333,14 @@ public static JsonObjectBuilder json(Dataset ds) {
                 .add("storageIdentifier", ds.getStorageIdentifier());
     }
 
+    public static JsonObjectBuilder json(fileDetailsHolder ds) {
```

Comment:
Given that this is globus specific, I'd suggest putting this function there (unless it is going to get called from elsewhere for some reason?). Also - any reason it isn't starting with a capital F?